### DISCUSSION TITLE
freebsd arm64

### DISCUSSION THREAD

**Discussion by celioreyes** [grafana-cold-storage/agent#1490] [Q&A]
Hello 👋🏽 ! 

I'm looking to compile the `agent` to use freebsd (arm64). Is there anything stopping arm64 support for freebsd from being added to the `agent`?

**Comment by rfratto** [comment]
👋 Official support would take a little bit of work, we currently use [rfratto/seego](https://github.com/rfratto/seego) to generate cross-compiled binaries for different platforms/architectures, and that doesn't have support for arm64 freebsd at the moment.

Off the top of my head, I don't think there's a reason arm64 freebsd builds wouldn't work, though.

**Reply by celioreyes** [reply]
Got it. I'm working on compiling it myself on our freebsd arm64 servers which took a bit of work but was able to get a binary built. Having issues with reading the `config.yml` right now. Going to be spending some time on getting it functional in the next few days
